src/dto/create-task.dto.ts

```diff
@@ -0,0 +1,14 @@
+import { IsEnum, IsNotEmpty, IsString } from 'class-validator';
+import { Severity } from '../task.types';
+
+export class CreateTaskDto {
+  @IsString()
+  @IsNotEmpty()
```

Comment:
```suggestion
  @IsString()
  @MinLength(1)
```
Вместо `@IsNotEmpty` лучше использовать `@MinLength(1)`